### ISSUE TITLE
can't compile python bindings

### ISSUE THREAD

**Issue by stormyweathers** [chronopoulos/libsequoia#1]
Hey there, just tried to build the python bindings and I get these three errors:

```
In file included from sequoiamodule.c:1:0:
sequoia-types.h:33:5: error: unknown type name ‘sq_port_t’
     sq_port_t port;
     ^
In file included from /home/user/miniconda3/include/python3.7m/Python.h:118:0,
                 from sequoia-types.h:1,
                 from sequoiamodule.c:1:
sequoiamodule.c: In function ‘initsequoia_worker’:
sequoiamodule.c:56:29: error: ‘PORT_IN’ undeclared (first use in this function)
     PyModule_AddIntMacro(m, PORT_IN);
                             ^

sequoiamodule.c:57:29: error: ‘PORT_OUT’ undeclared (first use in this function)
     PyModule_AddIntMacro(m, PORT_OUT);
```

I did make and install the C library, so those files are in my /usr/local/bin/ and /usr/local/include, which are both in my path.

Any idea what's going wrong?

**Comment by chronopoulos** [comment]
hey, thanks for your interest! there's been a lot of development on the C library recently, and the python bindings haven't been updated yet. the good news is, the C library is nearly at a good stopping point, so i'll probably tack over to the python side soon, and catch it up. thanks for opening the issue!

**Comment by chronopoulos** [comment]
fixed by 1ac664112c23af2d8c67a516cca1f20753fe5952
